docker-compose.yml

```diff
@@ -0,0 +1,152 @@
+version: "3"
+
+networks:
+  api-gateway:
+  loki:
+
+services:
+  gateway:
+    build:
+      context: .
+      dockerfile: ./nginx/Dockerfile.gateway
```

Comment:
Folder structure needs to changed, I guess. this docker file create Gateway image not nginx image